### ISSUE TITLE
Failed to train large images! 

### ISSUE THREAD

**Issue by opencomvis** [matterport/Mask_RCNN#202]
I have prepared my dataset with images and annotations, but when I get to training step "Exception: Invalid bounding box with area of zero" raised. my images are [ 1280 * 720] and I did all the annotation based on this size. 
how to fix the problem of large images, I see that the original paper used [800, 1024].
 Is there a way to modify and use large images instead?

**Comment by dexter1608** [comment]
@opencomvis hi I want to ask you some ques.
1. how did you prepared your data?
2. how did you loaded your data?
3. which annotation tool did you use?

**Comment by opencomvis** [comment]
@dexter1608 I already have my own dataset which has masks for regions of interest and to load data you need to modify the load_coco function to fit your data.. to annotate your data you need to read the annotations description for coco dataset first, then just apply those tips on your data.

**Comment by dexter1608** [comment]
@opencomvis 
https://github.com/cocodataset/cocoapi/issues/111 any suggestions?
thank you

**Comment by FumingX** [comment]
I have lots of 1280 * 1280 images on my own dataset.
Same as the #112 , I have some blank mask with all 0s.
By removing the blank masks, it works now.
Please double check your mask.

**Comment by fastlater** [comment]
@FumingX  how did you check your masks? Manually or you add some code lines somewhere in the script?

**Comment by FumingX** [comment]
@fastlater See #112 , @gaborvecsei gave a good method.

**Comment by kimile599** [comment]
> I have lots of 1280 * 1280 images in my own dataset.
> Same as #112 , I have some blank masks with all 0s.
> By removing the blank masks, it works now.
> Please double check your mask.

Hey, Do you maintain a same size for all training dataset? All of them 1280*1280? I just curious hows the training time
